generators/app/templates/src/layouts/guide.soy

```diff
@@ -24,16 +26,20 @@
  */
 {template .guide}
 	<div class="sidebar-offset">
+		{call topbar.render data="all" /}
```

Comment:
Hey @ygorcosta . I think we should find a way to do this without duplicating the topbar markup. Can we just use css/media queries to achieve the mobile navigation?